graphql/image_resolver.go

```diff
@@ -28,6 +28,34 @@ func (r *imageResolver) Distros(ctx context.Context, obj *model.APIImage) ([]*mo
 	return apiDistros, nil
 }
 
+// Events is the resolver for the events field.
+func (r *imageResolver) Events(ctx context.Context, obj *model.APIImage, limit int, page int) ([]*model.APIImageEvent, error) {
+	config, err := evergreen.GetConfig(ctx)
+	if err != nil {
+		return nil, InternalServerError.Send(ctx, fmt.Sprintf("getting evergreen configuration: '%s'", err.Error()))
+	}
+	c := thirdparty.NewRuntimeEnvironmentsClient(config.RuntimeEnvironments.BaseURL, config.RuntimeEnvironments.APIKey)
+	if obj == nil {
+		return nil, InternalServerError.Send(ctx, "image not provided")
+	}
```

Comment:
I'm going to contradict Bynn a little bit here. I think that we should either a) not check this like in all other resolvers or b) check it in all of the resolvers in this file. 

I don't think anything bad's going to happen if you exclude it (it's never checked in the patch, task, version, etc resolvers) and I think that's because the code exits early with an error if image turns out to be nil in the top-level resolver.

Reply:
that sounds fine to me